v5.0.0 is missing definitions

<!-- Requirements: please go through this checklist before opening a new issue -->

- [x] Review the documentation: https://docs.sentry.io/
- [x] Search for existing issues: https://github.com/getsentry/sentry-javascript/issues
- [x] Use the latest release: https://github.com/getsentry/sentry-javascript/releases

## Package + Version

- [x] `@sentry/browser`
- [ ] `@sentry/node`
- [ ] `raven-js`
- [ ] `raven-node` _(raven for node)_
- [ ] other:

### Version:

```
5.0.0
```

## Description

The TypeScript definitions for the latest version are missing. In v4.6.6 I see:
```
$ ls -l node_modules/@sentry/browser/dist/
total 904
-rw-r--r--   1 rgant  staff    1336 Oct 26  1985 backend.d.ts
-rw-r--r--   1 rgant  staff    1542 Oct 26  1985 client.d.ts
-rw-r--r--   1 rgant  staff    1750 Oct 26  1985 index.d.ts
-rw-r--r--   1 rgant  staff  164823 Oct 26  1985 index.js
-rw-r--r--   1 rgant  staff  261764 Oct 26  1985 index.js.map
drwxr-xr-x  10 rgant  staff     320 Apr  1 11:11 integrations
-rw-r--r--   1 rgant  staff      93 Oct 26  1985 md5.d.ts
-rw-r--r--   1 rgant  staff    1063 Oct 26  1985 parsers.d.ts
-rw-r--r--   1 rgant  staff    2893 Oct 26  1985 sdk.d.ts
-rw-r--r--   1 rgant  staff    3657 Oct 26  1985 tracekit.d.ts
drwxr-xr-x   6 rgant  staff     192 Apr  1 11:11 transports
-rw-r--r--   1 rgant  staff     107 Oct 26  1985 version.d.ts
```

In v5.0.0:
```
$ ls -l node_modules/@sentry/browser/dist/
total 920
-rw-r--r--  1 rgant  staff  184525 Oct 26  1985 index.js
-rw-r--r--  1 rgant  staff  281466 Oct 26  1985 index.js.map
```

When trying to build/run tests I get the following error:
```
ERROR in src/app/auth/auth.service.spec.ts(3,25): error TS7016: Could not find a declaration file for module '@sentry/browser'. '.../node_modules/@sentry/browser/dist/index.js' implicitly has an 'any' type.
  Try `npm install @types/sentry__browser` if it exists or add a new declaration (.d.ts) file containing `declare module '@sentry/browser';`
src/app/auth/auth.service.ts(3,25): error TS7016: Could not find a declaration file for module '@sentry/browser'. '.../node_modules/@sentry/browser/dist/index.js' implicitly has an 'any' type.
  Try `npm install @types/sentry__browser` if it exists or add a new declaration (.d.ts) file containing `declare module '@sentry/browser';`
src/app/auth/auth.service.ts(67,29): error TS7006: Parameter 'scope' implicitly has an 'any' type.
src/app/error-handler.spec.ts(3,25): error TS7016: Could not find a declaration file for module '@sentry/browser'. '.../node_modules/@sentry/browser/dist/index.js' implicitly has an 'any' type.
  Try `npm install @types/sentry__browser` if it exists or add a new declaration (.d.ts) file containing `declare module '@sentry/browser';`
src/app/error-handler.ts(2,25): error TS7016: Could not find a declaration file for module '@sentry/browser'. '.../node_modules/@sentry/browser/dist/index.js' implicitly has an 'any' type.
  Try `npm install @types/sentry__browser` if it exists or add a new declaration (.d.ts) file containing `declare module '@sentry/browser';`
```

--- a/.craft.yml
+++ b/.craft.yml
@@ -9,7 +9,7 @@ targets:
   - name: github
     includeNames: /^sentry-.*$/
   - name: gcs
-    includeNames: /*\.js.*$/
+    includeNames: /.*\.js.*$/
     bucket: sentry-js-sdk
     paths:
       - path: /{{version}}/

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## 5.0.1
+
+- [browser] fix: Add missing types
+
 ## 5.0.0
 
 This major bump brings a lot of internal improvements. Also, we extracted some integrations out of the SDKs and put them

--- a/packages/browser/package.json
+++ b/packages/browser/package.json
@@ -55,8 +55,9 @@
     "webpack": "^4.29.6"
   },
   "scripts": {
-    "build": "run-s build:esm build:bundle",
+    "build": "run-s build:dist build:esm build:bundle",
     "build:bundle": "rollup --config",
+    "build:dist": "tsc -p tsconfig.build.json",
     "build:esm": "tsc -p tsconfig.esm.json",
     "build:watch": "rollup --config --watch",
     "clean": "rimraf dist coverage .rpt2_cache build esm",

--- a/packages/browser/rollup.config.js
+++ b/packages/browser/rollup.config.js
@@ -66,18 +66,6 @@ const bundleConfig = {
 
 export default [
   {
-    input: 'src/index.ts',
-    output: {
-      file: 'dist/index.js',
-      format: 'cjs',
-      exports: 'named',
-      interop: false,
-      sourcemap: true,
-    },
-    external: ['tslib'],
-    plugins,
-  },
-  {
     ...bundleConfig,
     output: {
       ...bundleConfig.output,

